actions/campaigns.go

```diff
@@ -381,3 +381,61 @@ func GetCampaignOpens(c *gin.Context) {
 		"message": "Id must be an integer",
 	})
 }
+
+func GetCampaignStats(c *gin.Context) {
+	if id, err := strconv.ParseInt(c.Param("id"), 10, 64); err == nil {
+
+		var campaignStats entities.CampaignStats
+		campaignStats.TotalSent, err = storage.GetTotalSends(c, id)
+		if err != nil {
+			c.JSON(http.StatusNotFound, gin.H{
+				"message": "Campaign stats not found",
+			})
+			return
```

Comment:
currently returning in case any of stats is not found, I think we should not check the error. or return in case of internal error.

Reply:
This will usually return 0 in case there are no total sends right? Only a DB error would be thrown?